modules/nf-lineage/src/main/nextflow/lineage/LinObserver.groovy

```diff
@@ -259,17 +228,23 @@ class LinObserver implements TraceObserver {
             normalizer.normalizePath(task.getCondaEnv()),
             normalizer.normalizePath(task.getSpackEnv()),
             task.config?.getArchitecture()?.toString(),
-            task.processor.getTaskGlobalVars(task),
```

Comment:
Same as the TaskRun comment, I think it shouldn't be removed because it it used in the hash computation